### PR TITLE
changes to cli to integrate in overwrite commands

### PR DESCRIPTION
**Pull request by VoR0220**
This is for the epm suite so that we can implement an overwriting feature for jobs, with a default set to false.

### CODE REVIEW

File: cmd/pkgs.go [hunk 1]
```diff
@@ -81,6 +81,7 @@ func addPackagesFlags() {
 	packagesDo.Flags().StringVarP(&do.DefaultAddr, "address", "a", "", "default address to use; operates the same way as the [account] job, only before the epm file is ran")
 	packagesDo.Flags().StringVarP(&do.DefaultFee, "fee", "w", "1234", "default fee to use")
 	packagesDo.Flags().StringVarP(&do.DefaultAmount, "amount", "y", "9999", "default amount to use")
+	packagesDo.Flags().BoolVarP(&do.Overwrite, "overwrite", "t", false, "overwrite jobs of the same name")
```
Comment (zramsay) [inline cmd/pkgs.go]: is `-t` standard shortflag for `--overwrite` ?

Comment (VoR0220) [inline cmd/pkgs.go]: no...I would have put ```-w``` but it was already taken.

Comment (compleatang) [inline cmd/pkgs.go]: The default here should actually be true as epm isn't ran in interactive mode so won't actually be able to get the yes to the question when ran from cli.

**Comment by zramsay** [comment]
these commits can be squashed with `git rebase -i HEAD~3`
the `dont merge` labels is also still applied.

**Comment by zramsay** [comment]
LGTM

**Pull request merged**